ts/src/coinbaseinternational.ts

```diff
@@ -0,0 +1,1849 @@
+         * @param {string} id order id
+         * @param {string} symbol not used by coinbaseinternational cancelOrder()
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @returns {object} An [order structure]{@link https://docs.ccxt.com/#/?id=order-structure}
+         */
+        await this.loadMarkets ();
+        let portfolio = undefined;
+        [ portfolio, params ] = this.handleOptionAndParams (params, 'cancelOrder', 'portfolio');
+        if (portfolio === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelOrder() requires a portfolio parameter');
+        }
+        const request = {
+            'portfolio': portfolio,
+            'id': id,
+        };
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+        }
+        const orders = await this.v1PrivateDeleteOrdersId (this.extend (request, params));
+        //
+        //    {
+        //        "order_id":"1x96skvg-1-0",
+        //        "client_order_id":"ccxt",
+        //        "side":"BUY",
+        //        "instrument_id":"114jqr89-0-0",
+        //        "instrument_uuid":"b3469e0b-222c-4f8a-9f68-1f9e44d7e5e0",
+        //        "symbol":"BTC-PERP",
+        //        "portfolio_id":"1wp37qsc-1-0",
+        //        "portfolio_uuid":"018d7f6c-b92c-7361-8b7e-2932711e5a22",
+        //        "type":"LIMIT",
+        //        "price":"10000",
+        //        "size":"0.001",
+        //        "tif":"GTC",
+        //        "stp_mode":"BOTH",
+        //        "event_type":"CANCELED",
+        //        "order_status":"DONE",
+        //        "leaves_qty":"0.001",
+        //        "exec_qty":"0",
+        //        "avg_price":"0",
+        //        "fee":"0"
+        //    }
+        //
+        return this.parseOrder (orders, market);
+    }
+
+    async cancelAllOrders (symbol: string = undefined, params = {}) {
+        await this.loadMarkets ();
+        let portfolio = undefined;
+        [ portfolio, params ] = this.handleOptionAndParams (params, 'cancelOrder', 'portfolio');
+        if (portfolio === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelOrder() requires a portfolioId parameter');
+        }
+        const request = {
+            'portfolio': portfolio,
+        };
+        let market = undefined;
+        if (symbol) {
+            market = this.market (symbol);
+            request['instrument'] = market['id'];
+        }
+        const orders = await this.v1PrivateDeleteOrders (this.extend (request, params));
+        return this.parseOrders (orders, market);
+    }
+
+    async editOrder (id: string, symbol: string, type: OrderType, side: OrderSide, amount: number = undefined, price: number = undefined, params = {}) {
+        /**
+         * @method
+         * @name coinbaseinternational#editOrder
+         * @description edit a trade order
```

Comment:
we're missing the doc here https://docs.cloud.coinbase.com/intx/reference/modifyorder